### DISCUSSION TITLE
Sky cultures with extended with additional latin charaters

### DISCUSSION THREAD

**Discussion by vg5441** [Stellarium/stellarium#531] [Q&A] [answered]
Dear All, 

When I would like to extend Sky Cultures and star_names.fab with same line 

91262|_("Csősz - Vega") 1,2,6,11,12,13,18

on the sky I see

Cs?sz - Vega

I have the same problem when try to use same characters (á,é,í etc.) in case of planet names.

How can I get a proper "ő"  in this case?

Regards

**Comment by gzotti** [comment, marked as answer]
Use Unicode.

Is this for your private enjoyment, or are you preparing a contribution of some Hungarian skyculture?
In the latter case: Showing "foreign" and "regular" names at the same time in a useful way is something we still don't find time to discuss. Skycultures are going to change, but we don't know when.
If You just want to translate "Western" skyculture into Hungarian, please follow our translators' group on Transifex.

**Comment by vg5441** [comment]
Thanks for quick answer. On this level is private enjoyment only :-) ... can you give me little bit more information how can I able to use unicode in this case?

**Comment by gzotti** [comment]
You need a text editor which can store Unicode (UTF-8), e.g. Notepad++. Please Google for advice if you don't know what Unicode is.

**Comment by vg5441** [comment]
Thanks for help.
